plugins/unpack/lang/en.js

```diff
@@ -10,6 +10,7 @@
  theUILang.unpackPath		= "Unpack to (leave blank for the torrent's current directory)";
  theUILang.unzipNotFound	= "Unpack plugin: rTorrent user can't access 'unzip' program.";
  theUILang.unrarNotFound	= "Unpack plugin: rTorrent user can't access 'unrar' program.";
+ theUILang.7zNotFound		= "Unpack plugin: rTorrent user can't access '7z' program.";
```

Comment:
This is an error. JS identifier can't start with digit.
We should to have theUILang["7zNotFound"] form here.